plugins/destination/mysql/client/test_connection.go

```diff
@@ -8,21 +8,13 @@ import (
 	"github.com/rs/zerolog"
 )
 
-const (
```

Comment:
what's the reason for removing constants?

Reply:
I'd say it's better to use constants instead of duplicating strings in tests. However, if you feel differently, we can go ahead and merge it

Reply:
Readded in https://github.com/cloudquery/cloudquery/pull/18985/commits/588ddd7fac4a971d712faca6eb49c4fcfd134353